lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php

```diff
@@ -43,6 +43,9 @@ class ORMPurger implements PurgerInterface, ORMPurgerInterface
      */
     private $excluded;
 
+    /** @var string[]|null */
+    private static $cachedSqlStatements = null;
```

Comment:
I would not make it a static property. We can have multiple object managers, which will need different sql statements (different entities, different platforms). A global cache would require being keyed by something distinguishing those cases

Reply:
I removed static, because it is not required.
On project side I can define static variable with ORMPurger instance and reuse it